components/console-backend-service/internal/domain/oauth/oauth2client.go

```diff
@@ -0,0 +1,23 @@
+package oauth
+
+import (
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/resource"
+)
+
+type Resolver struct {
+	*resource.Module
+}
+
+func New(factory *resource.GenericServiceFactory) *Resolver {
+	module := resource.NewModule("apigateway", factory, resource.ServiceCreators{
```

Comment:
Shouldn't this be "oauth"

Reply:
It was originally (as changed in `Reuse BackendModule` commit), but if `oauth` and `apigateway` were to be defined always together, it makes sense to use the same module.